Say Their Names/Source/Controller/Home/HomeView.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  HomeView.swift
+//  Say Their Names
+//
+//  Created by Joseph A. Wardell on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+class HomeView: UIView {
+
+    var customNavBar: UIView!
```

Comment:
My suggestion would be to keep view configuration close to where it's declared. 
Then we can have init code separately to view setup code. This keeps initializers smaller and makes it easier to debug specific views. 

Also gets rid of the bang(!) 

```
lazy var customNavBar: UIView = {
let customNavBar = UIView()
customNavBar.backgroundColor = .black
return customNavBar
}
```

Reply:
I guess we should use the full name instead of abbreviated names. Instead of: `Nav`, use `Navigation`.

Reply:
to build up on @Mackis 's suggestion, we can even do 
```
let customNavBar: UIView = {
   let customNavBar = UIView()
   customNavBar.backgroundColor = .black
   return customNavBar
}()
```
to avoid unnecessary mutability.
The improvement can be done as a follow-up I think, just make sure to create a Trello card for it 🙏

Reply:
It's easy enough to do in this PR.  Thanks for the nudge.

Reply:
done